runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/WindmillStream.java

```diff
@@ -41,15 +40,13 @@ public interface WindmillStream {
   /** Handle representing a stream of GetWork responses. */
   @ThreadSafe
   interface GetWorkStream extends WindmillStream {
-    /** Functional interface for receiving WorkItems. */
-    @FunctionalInterface
-    interface WorkItemReceiver {
-      void receiveWork(
-          String computation,
-          @Nullable Instant inputDataWatermark,
-          @Nullable Instant synchronizedProcessingTime,
-          Windmill.WorkItem workItem,
-          Collection<Windmill.LatencyAttribution> getWorkStreamLatencies);
+
+    /** Adjusts the {@link GetWorkBudget} for the stream. */
+    default void adjustBudget(long itemsDelta, long bytesDelta) {}
```

Comment:
these don't seem like great defaults to have, can we just keep it abstract to make sure it is implemented? and put no-op in test impls if desired?

Reply:
done